cartographer_ros/cartographer_ros/node_options.cc

```diff
@@ -34,6 +37,14 @@ NodeOptions CreateNodeOptions(
       lua_parameter_dictionary->GetDouble("submap_publish_period_sec");
   options.pose_publish_period_sec =
       lua_parameter_dictionary->GetDouble("pose_publish_period_sec");
+  options.trajectory_publish_period_sec =
+      lua_parameter_dictionary->HasKey("trajectory_publish_period_sec")
+          ? lua_parameter_dictionary->GetDouble("trajectory_publish_period_sec")
+          : kDefaultTrajectoryPublishPeriodSec;
+  options.trajectory_hue_first =
```

Comment:
But they are already colored differently, as a different robots cannot be the same trajectory.